js/main.js

```diff
@@ -1,2 +1,52 @@
 /* global data */
 /* exported data */
+
+var $img = document.querySelector('div.column-half>img');
+var $form = document.querySelector('#journal-entry');
+
+function imgURL(event) {
+  if (event.target.getAttribute('name') !== 'imgURL') {
+    return;
+  }
+  var newURL = event.target.value;
+  $img.setAttribute('src', newURL);
+  if (event.target.value !== '') {
+    return;
+  }
+  $img.setAttribute('src', 'images/placeholder-image-square.jpg');
+}
+
+document.addEventListener('input', imgURL);
+
+function EntryFormData(event) {
```

Comment:
Please do not capitalize function names like this, it has a special meaning we will discuss in the future. camelCase, please